test/funcfile-docker-rt-tests/testfiles/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:latest
+RUN mkdir /app
+ADd . /app
```

Comment:
ADD

Reply:
Thanks for catch the case typo in 'ADd' as docker doc suggested docker command in Dockerfile should be upper case though it's not required.

Reply:
Should do upper case like everything else in all our Dockerfile's. At the very least, not random cased. ;)

Reply:
agreed